### PR TITLE
Update instrumentation-quickstart README

### PR DESCRIPTION
**Pull request by psx95**

### CODE REVIEW

**Review by dashpole — APPROVED** [review]

File: examples/instrumentation-quickstart/README.md [hunk 1]
```diff
@@ -39,11 +39,22 @@ oauth2 flow (read more about the command [here][auth_command]):
 
 	gcloud auth application-default login
 
-Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable with `export GOOGLE_APPLICATION_CREDENTIALS="$HOME/.config/gcloud/application_default_credentials.json"`
-or manually set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to a service
-account key JSON file path.
+**NOTE: This method of authentication is not recommended for production environments.**
```
Comment (aabmass) [inline examples/instrumentation-quickstart/README.md:42]: There is a GH markdown syntax for this https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

File: examples/instrumentation-quickstart/README.md [hunk 2]
```diff
@@ -39,11 +39,22 @@ oauth2 flow (read more about the command [here][auth_command]):
...
 
-Learn more at [Setting Up Authentication for Server to Server Production Applications][ADC].
+Executing this command will save your application credentials to default path which will depend on the type of machine -
```
Comment (aabmass) [inline examples/instrumentation-quickstart/README.md:44]:
```suggestion
Executing this command will save your application credentials to the default path which will depend on the type of machine -
```

File: examples/instrumentation-quickstart/README.md [hunk 3]
```diff
@@ -39,11 +39,22 @@ oauth2 flow (read more about the command [here][auth_command]):
...
+- Linux, macOS: `$HOME/.config/gcloud/application_default_credentials.json`
+- Windows: `%APPDATA%\gcloud\application_default_credentials.json`
+
+Next, export the credentials to `GOOGLE_APPLICATION_CREDENTIALS` environment variable -
+
+For Linux & MacOS:
```
Comment (aabmass) [inline examples/instrumentation-quickstart/README.md:50]: Should we include how for windows too?

File: examples/instrumentation-quickstart/README.md [hunk 4]
```diff
@@ -39,11 +39,22 @@ oauth2 flow (read more about the command [here][auth_command]):
...
+```shell
+export GOOGLE_APPLICATION_CREDENTIALS=$HOME/.config/gcloud/application_default_credentials.json
+```
+
+You can also manually set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to a service account key JSON file path.
+
+> Learn more at [Setting Up Authentication for Server to Server Production Applications][ADC].
```
Comment (aabmass) [inline examples/instrumentation-quickstart/README.md:57]: nit remove quoting here
```suggestion
Learn more at [Setting Up Authentication for Server to Server Production Applications][ADC].
```

**Review by aabmass — COMMENTED** [review]
Would you mind sending a PR making these same updates here? https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/tree/main/samples/instrumentation-quickstart

**Review by aabmass — APPROVED** [review]

**Review by aabmass — APPROVED** [review]

**Comment by psx95** [comment]
> Would you mind sending a PR making these same updates here? https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/tree/main/samples/instrumentation-quickstart

Sure, I will send a PR

**Pull request merged**